tests/test_utils.py

```diff
@@ -374,6 +377,53 @@ def test_convert_to_b64_no_overwrite(converter):
                 os.remove(target)
 
 
+@pytest.fixture
+def patch_file_ops():
+    with patch("connectors.utils.open"):
+        with patch("os.remove"):
+            with patch("os.rename"):
+                yield
+
+
+@pytest.mark.parametrize("converter", ["system"])
```

Comment:
Is there a way to [parametrize](https://docs.pytest.org/en/7.1.x/example/parametrize.html) these 3 tests? They look the same just `platform.mac_ver` and `platform.system` change.

Reply:
Good call, parametrized!